tests/unittests/BackendTestUtils.h

```diff
@@ -20,6 +20,16 @@
 
 namespace glow {
 
+/// MockBackend used only for unit testing.
+class MockBackend : public Backend {
```

Comment:
there is no immediate need, but would be better to make a real gtest mock, so that we could plug expect_call and behavior on specific methods.
This way we can test if shouldLower (and others) were actually called and enforce specific behavior if needed.